.pipelines/containerSourceData/valkey/configuration-files/valkey-docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Is this entrypoint script the upstream expected behavior for valkey container?

Reply:
Thank you for pointing it out. There are certain additions like "VALKEY_EXTRA_FLAGS" parameter in upstream [[link](https://github.com/valkey-io/valkey-container/blob/7f43f22530d6eaf017d9e95824c6a4a9767755a2/8.0/alpine/docker-entrypoint.sh#L22)]. Let me check that and add here if required.